### PR TITLE
ci: #861mhn6b6: move to bitnami chart

### PR DESCRIPTION
**Pull request by msipatov**
Delete old chart template and replace it with new chart and configure it

### CODE REVIEW

File: .github/workflows/dev-docker-publish.yml [hunk 1]
```diff
@@ -31,20 +31,20 @@ jobs:
...
-            --set "imageCredentials.password=${{ secrets.GH_REGISTRY_TOKEN }}"
-            --set "image.tag=${{ github.sha }}"
-            --set "image.pullPolicy=Always"
-            ./SalaryService.Api/ci/chart
+          exec: |
+            RELEASE_NAME=dev-inner-circle-salary-service-api
+            helm repo add bitnami https://charts.bitnami.com/bitnami
+            helm upgrade --install --namespace dev-inner-circle --create-namespace --values ./SalaryService.Api/ci/values-custom.yaml \
+            --set "image.pullSecrets={${RELEASE_NAME}-aspnet-core-docker-secret}" \
+            --set "image.pullCredentials.username=${{ github.actor }}" \
```
Comment (vvvictor07) [inline .github/workflows/dev-docker-publish.yml:39]: It seem that it works wrong for now. You need to create username and token for registry access in the ci secrets or use something like [this](https://docs.github.com/en/actions/security-guides/automatic-token-authentication) (**Note:** but it only work for build, not for deploy)

File: SalaryService.Api/ci/values-custom.yaml [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+args:
+  - "SalaryService.Api.dll"
+  # - "EmailSender.Api.dll"
+
+## @param bindURLs URLs to bind
+##
+bindURLs: http://+:80
+
+# Here stored regular vars that passed to the pod environment
+extraConfigMapEnvVars:
```
Comment (vvvictor07) [inline SalaryService.Api/ci/values-custom.yaml:47]: Could we have here default value for dictionary

Comment (msipatov) [reply SalaryService.Api/ci/values-custom.yaml]: Added default value for this variable

**Review by vvvictor07 — APPROVED** [review]
All is ok

**Pull request merged**